docs/latest/user/grpc-routing.md

```diff
@@ -90,6 +90,52 @@ Envoy Gateway also supports [gRPC-Web][] requests for this configuration. The be
 curl --http2-prior-knowledge -s ${GATEWAY_HOST}:80/yages.Echo/Ping -H 'Host: grpc-example.com'   -H 'Content-Type: application/grpc-web-text'   -H 'Accept: application/grpc-web-text' -XPOST -d'AAAAAAA=' | base64 -d
 ```
 
+## GRPCRoute Match
+The `matches` field can be used to restrict the route to a specific set of requests based on GRPC's service and method names.
+The following example shows how to match requests for any method in the `yages.Echo` service and requests for the `ServerReflectionInfo` method in any service.
+
+```shell
+cat <<EOF | kubectl apply -f -
+apiVersion: gateway.networking.k8s.io/v1alpha2
+kind: GRPCRoute
+metadata:
+  name: yages
+  labels:
+    example: grpc-routing
+spec:
+  parentRefs:
+    - name: example-gateway
+  hostnames:
+    - "grpc-example.com"
+  rules:
+    - matches:
+      - method:
+          method: ServerReflectionInfo
```

Comment:
shouldnt this be `Ping` ?

Reply:
in this case I'm matching for another service/method - grpc.reflection.v1alpha.ServerReflection/ServerReflectionInfo, but I see how this can be confused as an example. I'll change it to match for method and service

Reply:
if the example worked with existing config and a grpc call to `yages.Echo/Ping`, it makes me think that the regex string config  is incorrect

Reply:
existing example matches a request to any service that has method `ServerReflectionInfo`(will not match `yages.Echo/Ping` ) OR a request to any method in a service `yages.Echo` which matches ` yages.Echo/Ping`. We need to match both because without matching ServerReflectionInfo you will get `Error invoking method "yages.Echo/Ping": failed to query for service descriptor "yages.Echo": server does not support the reflection API`. Probably cleaner way will be to exact match both methods and services, but I wanted to showcase Exact and RegularExpression types of matches.

Reply:
thanks for clarifying that ! the `exact` match makes more sense here imho